### PR TITLE
Handbook update major announcements & reward program

### PR DESCRIPTION
**Pull request by w1l1ams**

### CODE REVIEW

**Comment by w1l1ams** [comment]
Done making edits. This can be merged

File: marketing/README.md [hunk 1]
```diff
@@ -4,39 +4,62 @@
...
 #### Minor announcements
 
-1. Post on [Telegram](https://t.me/mesg_community)
-2. Post on [Sendible](https://app.sendible.com)
-  - Brief announcement
-  - Post to Twitter, LinkedIn and Facebook
+1. Create event on [Google Calendar](https://calendar.google.com/) using the MESG Social account
+  - Ensure date and time of future post are correct
+  - Write post in the description box of the event 
+  - Check that links are not automatically underlined/made clickable by Google. To do this, avoid hitting 'space' after link is pasted and/or toggle off the link button.
```
Comment (antho1404) [inline marketing/README.md]: this will need to be updated as well as this \ for new line hack

Comment (w1l1ams) [reply marketing/README.md]: Sounds good - I'll add it once we get it all set up and tested.

**Comment by w1l1ams** [comment]
Ready to close!

File: marketing/README.md [hunk 2]
```diff
@@ -4,39 +4,71 @@
...
+  - Ensure date and time of future post are correct
+  - Write post in the description box of the event  
+  - Use a forward slash (/) to add a line break separating the URL from the hashtags
+  - Duplicate post to on all calendars: Twitter, Linkedin, Facebook, Discord and Telegram
+  - Edit posts per platform, as necessary
+2. Bi-Weekly News is posted on Medium once every two weeks, summarizing developments over the past two weeks 
+  - Include categories for: Research and Development, Marketing & Community, and Partners & Administration 
+  - Link to mentioned content as much as possible
+  - Include a visual to break up the post
+  - After publishing article on the MESG Publication, do not choose checkbox to allow others to recommend story for Medium paywall
```
Comment (antho1404) [inline marketing/README.md]: Why not? We can always have it but still make sure that we use the "link for a friend" in our content

File: marketing/README.md [hunk 3]
```diff
@@ -4,39 +4,71 @@
...
-  - Pin the announcement
-4. Post on [Sendible](https://app.sendible.com)
-  - Brief announcement
-  - Link to the blog post
-  - Post to Twitter, LinkedIn and Facebook
-5. Send post on [Discord](https://discordapp.com/invite/SaZ5HcE)
-  - Include @everyone
-6. Send a newsletter via [Sendgrid](https://sendgrid.com/)
+1. Post article on [Medium](https://medium.com/mesg) Blog
+  - After publishing article on the MESG Publication, choose checkbox to allow others to recommend story for Medium paywall
```
Comment (antho1404) [inline marketing/README.md]: add a part to copy the link from the "link for a friend" and not the url

File: marketing/README.md [hunk 4]
```diff
@@ -4,39 +4,71 @@
...
+  - Ensure date and time of future post are correct
+  - Write post in the description box of the event 
+  - Use a forward slash (/) to add a line break separating the URL from the hashtags
+  - Duplicate post to on all calendars: Twitter, Linkedin, Facebook, Discord and Telegram
+3. Send a newsletter via [Sendgrid](https://sendgrid.com/)
   - Brief announcement 
   - Accompanying graphic/image
   - Link to the blog post
   
+#### Major product releases
```
Comment (antho1404) [inline marketing/README.md]:
```suggestion
#### Major product feature
```

Let's announce feature not releases. A feature might be split into multiple releases

File: marketing/README.md [hunk 5]
```diff
@@ -4,39 +4,71 @@
...
-1. 3 to 5 social media posts per week should be scheduled using [Sendible](https://app.sendible.com)
-  - All content is to be posted to Twitter, Facebook and Linkedin
-  - Scheduled posts can be reposts, where content is reposted 4 or 5 times after the original post
-  - Rewrite post and link article 
+1. Three to five social media posts per week should be scheduled using [Google Calendar](https://calendar.google.com/)
+  - Create event using the MESG Social account
+  - Ensure date and time of future post are correct
+  - Write post in the description box of the event
+  - Use a forward slash (/) to add a line break separating the URL from the hashtags
+  - Ensure links are not automatically underlined/made clickable by Google. To do this, avoid hitting 'space' after link is pasted and/or toggle off the link button.
```
Comment (antho1404) [inline marketing/README.md]: not a problem anymore

File: marketing/README.md (hunk 2 above)
Comment (w1l1ams) [reply marketing/README.md]: fixed 👍

File: marketing/README.md (hunk 3 above)
Comment (w1l1ams) [reply marketing/README.md]: done

**Comment by w1l1ams** [comment]
Ok great - Changes made 👍

**Pull request merged**